config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -1330,7 +1330,7 @@ periodics:
           - --gcp-project-type=node-e2e-project
           - --gcp-zone=us-west1-b
           - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/containerd/image-config.yaml
-          - --node-test-args=--kubelet-flags="--exit-on-lock-contention --lock-file=/var/run/kubelet.lock"
+          - '--node-test-args=--container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--exit-on-lock-contention --lock-file=/var/run/kubelet.lock --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}"'
```

Comment:
Hi @ipochi - do we need these two flags (i.e.: `--exit-on-lock-contention --lock-file=/var/run/kubelet.lock` ) since they are not present in any other containerd job.

Reply:
@Namanl2001 yes, please check the PR description and rest job config(line:1336), this job is testing the feature `lock contention` so yes it is required